ccpp/suites/suite_FV3_GFS_2017_fv3wam.xml

```diff
@@ -0,0 +1,86 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<suite name="FV3_GFS_2017_fv3wam" lib="ccppphys" ver="3">
```

Comment:
What is the difference of this file and the FV3_GFS_2017?

Reply:
The difference between two files. Only no NSST in fv3wam.
Should ver="4" for ccppphys instead of "3"? Will it impact results? 

< <suite name="FV3_GFS_2017_fv3wam" lib="ccppphys" ver="3">
---
> <suite name="FV3_GFS_2017" lib="ccppphys" ver="4">
42c42,44
<       <scheme>sfc_ocean</scheme>
---
>       <scheme>sfc_nst_pre</scheme>
>       <scheme>sfc_nst</scheme>
>       <scheme>sfc_nst_post</scheme>
63d64
<       <scheme>GFS_DCNV_generic_pre</scheme>
65a67
>       <scheme>GFS_DCNV_generic_pre</scheme>

Reply:
It should be 4 as it is the current version CCPP in fv3atm.